cli/src/main/java/org/opentosca/toscana/cli/commands/Constants.java

```diff
@@ -8,65 +8,34 @@
     public static final String PARAM_CSAR = "<CSAR>";
     public static final String PARAM_PLATFORM = "<Platform>";
     public final Integer API_PORT = 8084;
-    public final String CSAR_UPLOAD_SUCCESS = "Upload of CSAR Archive was successful!";
-    public final String CSAR_UPLOAD_ERROR = "Something went wrong while uploading the CSAR.";
-    public final String CSAR_UPLOAD_ERROR400 = "Parsing of the CSAR failed.";
-    public final String CSAR_UPLOAD_ERROR400M = "A CSAR with given name already exists.\n";
-    public final String CSAR_UPLOAD_ERROR500 = "Processing failed.\n";
-    public final String CSAR_DELETE_SUCCESS = "Deletion of CSAR was successfull!";
-    public final String CSAR_DELETE_ERROR = "Something went wrong while deleting the CSAR.";
-    public final String CSAR_DELETE_ERROR404 = "CSAR not found!";
-    public final String CSAR_DELETE_ERROR500 = "Deletion of CSAR has failed!";
-    public final String CSAR_LIST_SUCCESS = "List of CSARs:";
+    public final String CSAR_UPLOAD_IO_ERROR = "Something went wrong while loading Csar:";
```

Comment:
Please consider making these fields static

Reply:
Did you realize that you use "Something went wrong while <action> <artifact>:" the whole time? This can be shortened a lot.